a.core_concepts.md

```diff
@@ -130,7 +136,7 @@ kubectl create namespace myns -o yaml --dry-run
 <p>
 
 ```bash
-kubectl create resourcequota myrq -o yaml --dry-run
+kubectl create quota myrq --hard=cpu=1,memory=1G,pods=2,services=3,secrets=5,persistentvolumeclaims=10 --dry-run -o yaml
```

Comment:
Good point, I'd write that as 
```bash
kubectl create quota myrq --hard=cpu=1,memory=1G,pods=2 --dry-run -o yaml
```
I think it's simpler :)

Reply:
agreed